common/metrics/metric_defs.go

```diff
@@ -27,6 +27,7 @@ const (
 	resourceExhaustedScopeTag   = "resource_exhausted_scope"
 	PartitionTagName            = "partition"
 	PriorityTagName             = "priority"
+	PersistenceDBKindTagName    = "db_kind"
```

Comment:
Why do users need the db_kind tag? They should know what DB their cluster is running.

Reply:
because temporal maintain separated sql pool for primary and visibility db. In the scenario of both primary and visibility using sqlplugin, without `db_kind` tag would result in merged total count of connection from both primary and visibility cluster

Reply:
Ah, thanks!